examples/seismic/preset_models.py

```diff
@@ -219,7 +219,7 @@ def demo_model(preset, **kwargs):
                              dtype=dtype, spacing=spacing, nbl=nbl, epsilon=epsilon,
                              delta=delta, theta=theta, phi=phi, bcs="damp", **kwargs)
 
-        if kwargs.get('smooth', True):
+        if kwargs.get('smooth', False):
```

Comment:
why?

do we now get smoothing by default? when I run `python tti_example.py` for example

Reply:
Yes that's what ` kwargs.get('smooth', True)` does, currently needs to explicitly ask for no smoothing and not easy in bench/examples (need extra cli arg)